cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -80,7 +80,9 @@ def _refresh_children(self, lib_content_block, status_code_expected=200):
         of a LibraryContent block
         """
         if 'user' not in lib_content_block.runtime._services:  # pylint: disable=protected-access
-            lib_content_block.runtime._services['user'] = Mock(user_id=self.user.id)  # pylint: disable=protected-access
+            mocked_user_service = Mock(user_id=self.user.id).get_current_user.return_value = {}
```

Comment:
I think this is going to assign `{}` to `mocked_user_service`. Is that what you intended?